miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -25,6 +28,11 @@ abstract class MiniAppMessageBridge {
         }
     }
 
+    @JavascriptInterface
+    fun shareTo(content: String) {
```

Comment:
This is okay for POC purposes, but for the actual implementation `shareTo` should be passed through `postMessage` and included in the JS SDK.